xtask/libs_bench/benches/contains_criterion.rs

```diff
@@ -4,12 +4,23 @@ use contains::*;
 use criterion::{criterion_group, criterion_main, Criterion};
 use fastbloom_rs::Membership;
 
+const SEARCH_FOR: &[&str] = &[
+    "undefined",
+    "a",
+    "NaN",
+    "longVariableName",
+    "Infinity",
+    "xxxxxxxx",
+    "arguments",
+    "eval",
+];
+
```

Comment:
This was probably a function originally to prevent the compiler from optimizing the loop in these benchmarks. Could be wrong though, I'd like someone else to weigh in.